tools/c7n_azure/tests/test_session.py

```diff
@@ -47,6 +48,19 @@ def test_initialize_session_principal(self):
                 self.assertIs(type(s.get_credentials()), ServicePrincipalCredentials)
                 self.assertEqual(s.get_subscription_id(), 'ea42f556-5106-4743-99b0-c129bfa71a47')
```

Comment:
You can use DEFAULT_SUBSCRIPTION_ID for the subscription id, it comes from azure_common.